components/pageDescription/PageDescription.js

```diff
@@ -0,0 +1,22 @@
+import { getString } from '../../lib/richText';
+import {
+  PageDescriptionContainer,
+  DescriptionContainer,
+  Description,
+  Img,
+} from './PageDescription.styles';
+import LongSparkArrow from '../longSparkArrow/LongSparkArrow';
+
+export default function PageDescription({ description, arrowText, img }) {
+  return (
+    <div>
+      <PageDescriptionContainer>
+        <LongSparkArrow arrowText={arrowText} onClick={() => {}} />
```

Comment:
why is `onClick` used here?

Reply:
Not sure why I passed it an empty function (just fixed) but the onClick is there for in the future when we implement the clicking animation.

Reply:
Can you remove it for the moment if it's not being used please?